Local devdelopment with examples (#1330)
vercel/swr: CONTRIBUTING.md

## Open a PR for Bugfix or Feature

> ### Local development with examples

Reviewer: ```suggestion
### Local Development with Examples
```

> To run swr locally, you can develop it with any example in `examples` folder. You need to setup the example and run command in root directory for overriding swr and its dependencies to local assets.

Reviewer: ```suggestion
To run SWR locally, you can start it with any example in `examples` folder. You need to setup the example and run command in the root directory for overriding SWR and its dependencies to local assets.
```

> First of all, build swr assets

Reviewer: ```suggestion
First of all, build SWR assets:
```

```sh
# or `yarn watch`
yarn build
```

> Install dependency of the target example, for instance `examples/basic`

Reviewer: ```suggestion
Install dependency of the target example, for instance `examples/basic`:
```

```sh
cd examples/basic && yarn
```

> After setup, back to the root directory and run

Reviewer: ```suggestion
After setup, back to the root directory and run:
```

```sh
# by default it will run next dev for the example
yarn dev-next basic
```

> All examples are built with next.js, so the extract next command is supported

Reviewer: ```suggestion
All examples are built with Next.js, so Next.js commands are all supported:
```
